tests/helpers.py

```diff
@@ -1369,17 +1370,63 @@ def verify_volume_deleted_in_backend(interface, image_uuid, pool_name=None):
 
     try:
         ct_pod.exec_ceph_cmd(ceph_cmd=cmd, format='json')
-        return False
+        logger.info(
+            f"Verified: Volume corresponding to uuid {image_uuid} exists "
+            f"in backend"
+        )
+        return True
     except CommandFailed as ecf:
         assert valid_error in str(ecf), (
-            f"Error occurred while verifying volume is deleted in backend: "
+            f"Error occurred while verifying volume is present in backend: "
             f"{str(ecf)} ImageUUID: {image_uuid}. Interface type: {interface}"
         )
-    logger.info(
-        f"Verified: Volume corresponding to uuid {image_uuid} is deleted "
-        f"in backend"
-    )
-    return True
+        logger.info(
+            f"Volume corresponding to uuid {image_uuid} does not exist "
+            f"in backend"
+        )
+        return False
+
+
+def verify_volume_deleted_in_backend(
+    interface, image_uuid, pool_name=None, timeout=180
+):
```

Comment:
As per discussion within Green squad, let's try to make use of  `ceph rbd task list` or `ceph progress`  to see any pending actions of background image removal instead of random timeout.

Reply:
Added.